spec.md

```diff
@@ -22,21 +22,21 @@ log](https://git.fsfe.org/reuse/docs/src/branch/stable/CHANGELOG.md).
 
 These are the definitions for some of the terms used in this specification:
 
-- REUSE Tool --- helper tool for compliance with this Specification; available
+- REUSE Tool --- helper tool for compliance with this specification; available
   at <https://github.com/fsfe/reuse-tool>.
-
 - Project --- any unit of content that can be associated with a distribution of
   software. Typically, a Project is composed of one or more files. Also
   sometimes called a package.
-
 - License File --- a file containing the text of a license.
-
 - Copyright and Licensing Information --- the information that lists the
   copyright holders of a file or work, and describes under which licenses the
   file or work is made available.
-
+- Copyright Notice --- a line of text that conveys copyright of a copyright
+  holder. Its format is defined in the [Format of Copyright
+  Notices](#format-of-copyright-notices) section.
 - Covered File --- any file in a Project, except for
     - The License Files.
+    - `COPYING` and `LICENSE`, with or without file extensions.
```

Comment:
If we're already specific here, what about mentioning `LICENSES/` here or in the line above?

Reply:
agree